Shop/shop_item_torch.gd

```diff
@@ -0,0 +1,33 @@
+extends ShopItem
+
+@export var clickHandler : ClickHandler
+var torchTexture : PackedScene = preload("res://Shop/tourchDisplay.tscn")
```

Comment:
why is this not `@export` ? also it's `tourchDisplay` it should be renamed `torchDisplay` :laughing: